.github/workflows/github-actions-main.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           sudo apt-get install -y libsecret-1-dev
           sudo apt-get install -y translate-toolkit
           sudo apt-get install -y rsync
+          sudo apt-get install -y xvfb
```

Comment:
Could you add a comment so that we know what depends on this package please? How about developing on Windows or macOS - do we need to install something? In which case that should be documented in BUILD.md

Reply:
The tests run Windows and MacOS without additional dependencies. `xvfb` provides a virtual X11 display, so is only needed on Linux systems without a display server, as seems to be the case on GitHub CI.